app/controllers/users_controller.rb

```diff
@@ -1,3 +1,27 @@
 class UsersController < ApplicationController
-  def new; end
+  attr_reader :user
+  def show
+    @user = User.find(params[:id])
```

Comment:
@tbvietto nếu trong database ,ko có User nào có id bằng với params[:id] thì hàm find sẽ bung ra một lỗi ngay lập tức, thậm chí nếu khắc phục dc việc bung lỗi thì @user cũng sẽ dc gán giá trị thành nil, dẫn đến bất kì đoạn code @user.any_method nào cũng bị lỗi( ví dụ làm gì có trò nil.id ^^). Vì thế nên kiểm tra trg hợp @user ==nil trc khi thực hiện các code tiếp theo trong action show đấy. Nên làm như thế này:
def show
   @user=User.find_by_id(params[:id])
   if @user #tức là nếu @user khác nil
      bla bla 
   else 
      flash[:danger]='Not found'
      redirect_to root_path
   end 
end